### ISSUE TITLE
[Feature Request]: (at) somebody to explicitly notify and address someone

### ISSUE THREAD

**Issue by stevenleeS0ht** [iNPUTmice/Conversations#4112]
Just like any other instant messaging and this GitHub message (@iNPUTmice ). 

Currently if you tap someone's avatar, it will add `username:` in the textbox. 

However, it is not comfortable and convenient. It would be nice if the behaviors is like below:

when input `@` in the textbox (single character), there will be a droplist which list all candidate username.

When keep input the initial characters after `@`, the droplist will become narrow and narrow. The available items in the droplist are sorted in numalpha order.

**Comment by Apichat-fr** [comment]
Hello I've write some though on how the name should be manage on Internet, you can read it here : 
http://apichat.fr/doku.php?id=hello_joe_greeting_editorial_rule_and_name_on_the_internet

TLDR
the semantic username@hostname (which is promote since 1982 in RFC 805 about email) is really bad for user's freedom and profits to the centralized social platforms.

We should use greeting@name like hello@jdoe.net read my text to learn more on the interest of the greeting part.

Suddenly, on a user interface, when the relation is already established, to notify and address someone the greeting part could be hidden (or not) : @jdoe.net

And to get decentralized Internet names Gnu Name System (GNS) will be a good tool : 
- https://gnunet.org/en/
- https://www.ngi.eu/event/ngi-projects-contribution-to-technological-developments-of-dns-and-naming-systems/
